Svenbjorn/whatson
# added animation to menu display and language item in menu

Added animation to menu display and to language menu item

TO TEST
- open and close the menu, verify that it behaves as it does in figma: https://www.figma.com/proto/eOshLuKuFSfE7gWDYgVcFb/VFI-Tourism---mobile?node-id=0-1971&scaling=min-zoom&page-id=0%3A736&starting-point-node-id=0%3A1971&show-proto-sidebar=1
- Ignore the hamburger icon jump in Figma, this is not intentional
- Ignore the animation from hamburger to X. This requires more work than makes sense to make the transition look worse
- With the menu open, press the languages menu point and verify that it opens with animation shown in figma. Use the VFI menu as reference (height animation), since the whatson menu (fade animation) has not been completed on the Figma side.
- The animation trigger button (chevron) has not been included in this PR. This is because this is a global button, which should be developed as part of style changes. This button has not been added in the whatson menu on Figma
